internal/services/authorization/role_assignment_resource.go

```diff
@@ -413,3 +414,20 @@ func getTenantIdBySubscriptionId(ctx context.Context, client *subscriptions.Clie
 	}
 	return *resp.TenantID, nil
 }
+
+func normalizeScopeValue(scope string) (result string) {
+	if rg, err := commonids.ParseResourceGroupIDInsensitively(scope); err == nil {
+		return rg.ID()
+	}
+
+	if sub, err := commonids.ParseSubscriptionIDInsensitively(scope); err == nil {
+		return sub.ID()
+	}
+
+	if group, err := commonids.ParseManagementGroupIDInsensitively(scope); err == nil {
+		return group.ID()
+	}
```

Comment:
we should only need to normalize the resourcegroup here

Reply:
Updated.